### PR TITLE
[Anel] Compatibility with Anel Firmware 6.1.

### LINKED ISSUE #5262: [anel] Support for anel hut 6.1
## Expected Behavior
As an owner of an anel hut, I want to upgrade my anel HUT to 6.1 and OpenHab be able to still talk to it.
Preferable also using the newly added features for dimmers (the Hold On functionality)
(And to make it explicit, I would want it to support Anel hut 6.0 and 6.1 at the same time, so I can test the now no longer working 6.1 before I upgrade all my hut's to 6.1) 

## Current Behavior
After an upgrade of an anel Hut to 6.1, OpenHab can no longer talk to it.

## Possible Solution
The new api is described here https://anel-elektronik.de/forum_neu/viewtopic.php?f=16&t=207

## Steps to Reproduce (for bugs)
upgrade an anel hut to 6.1

## Your Environment
I had this problem with openhab 2.0 and OpenHab 2.1
(and as this is because of an upgrade of Anel hut, I suspect this problem exists in OpenHab 1.x also.

### PR DESCRIPTION
**Pull request by paphko**
Fixes #5262

### CODE REVIEW

File: bundles/binding/org.openhab.binding.anel/src/main/java/org/openhab/binding/anel/internal/AnelDataParser.java [hunk 1]
```diff
@@ -130,6 +144,11 @@
             }
         }
 
+        // since Firmware 6.0
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.anel/src/main/java/org/openhab/binding/anel/internal/AnelDataParser.java]: Remove this empty code block.

**Comment by yveshanoulle** [comment]
From what @paphko wrote, I assume it's fixed now.
Yet I'm not sure how I can test this. I assume I cant' right now, waiting for another code change. 
Is this correct @9037568 ?

**Comment by 9037568** [comment]
Correct. You'd have to either build it yourself or get @paphko to provide you a test jar.

**Comment by yveshanoulle** [comment]
I don't have a java editor on my machine. 
As Im' in the middle of building a house, I don't have the time either to set this up properly I'm afraid that getting OpenHab working on dev for the first time, will cost me a lot of time before the current version might work. 
@paphko will you send me the jar and tell me in what folder I should put it. 
(and whatever that is important to do that..) 

Yves

**Comment by paphko** [comment]
Sorry, I don't have an up-to-date IDE either at the moment.

Usually it should be possible to get the jar-file from Jenkins, but the corresponding job is currently offline: https://openhab.ci.cloudbees.com/job/PR-openHAB1-Addons/

@kaikreuzer: is there a reason for Jenkins to not build the OH1 add ons any longer? It says: "Pull Request Builder for openHAB1 Addons. Dieses Projekt ist momentan deaktiviert."

**Comment by kaikreuzer** [comment]
@paphko - I cannot remember. I just activated it again now, let's see what happens :-)

**Comment by kaikreuzer** [comment]
Looks successful :-)

**Comment by paphko** [comment]
Thank you @kaikreuzer !

@yveshanoulle here you can download the jar: https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/binding/org.openhab.binding.anel/1.11.0-SNAPSHOT/org.openhab.binding.anel-1.11.0-SNAPSHOT.jar

File: bundles/binding/org.openhab.binding.anel/src/main/java/org/openhab/binding/anel/internal/AnelDataParser.java (hunk 1 above)
Comment (paphko) [inline bundles/binding/org.openhab.binding.anel/src/main/java/org/openhab/binding/anel/internal/AnelDataParser.java]: Done.

**Comment by yveshanoulle** [comment]
@paphko I have never installed OpenHab 2.0 manually. What is the location to drop this file? 
Are their caveats on installing this manually? 
Is this now part of the latest daily build I can install?

**Comment by paphko** [comment]
@yveshanoulle I still have OH1 running and I cannot remember how OH2 deals with OH1 addons. Moreover, this discussion is out of scope for this issue. Please search the [docs](http://docs.openhab.org/tutorials/migration.html#installing-unofficially-supported-openhab-1x-add-ons) or use the [forum](https://community.openhab.org/) to ask this question.

**Comment by yveshanoulle** [comment]
the binding is no longer broken. I can talk to an anel hut 6.1. and to an anel hut 6.0

The new functionality in the anel hut 6.1, that is the dimmer functionality does not seem to work.
For me this fix is already a lot better then the previous version, as it's no longer broken. 

so I think this could be released. 

I'll look on thursday evening to see if I can add a new bug for the new functionality and explain what is happening and what is supposed to happen.

**Comment by 9037568** [comment]
Thanks @paphko and @yveshanoulle for your work on this.

**Pull request merged**